cfgmgr/intfmgr.cpp

```diff
@@ -13,6 +13,7 @@ using namespace swss;
 
 #define VLAN_PREFIX         "Vlan"
 #define LAG_PREFIX          "PortChannel"
+#define VNET_PREFIX         "Vnet"
 
```

Comment:
can you add vs test for this refactoring.

I am think you need to first create 

INTERFACE|eth0 with vrf, then check if the interface has been moved to vrf or not.

INTERFACE|eth0|ipaddress, then check if the interface has been assigned with IP address or not.

add another ip address, and check if the interface has been assigned with additional IP.

then, remove the one ip address, and check.

remove another ip address, and check.

Then, check vrf name and check if the interface is moved to another vrf or not.